client/src/components/relatedProducts/card.jsx

```diff
@@ -29,15 +32,17 @@ const CardParagraphs = styled.p`
 `
 
 const Card = (props) => {
-  axios
 
   return (
     <Carta>
-    <Pic src="https://image.shutterstock.com/image-vector/colorful-illustration-test-word-260nw-1438324490.jpg" />
+    <Pic src={props.picUrl} />
       <Container>
-      <CardParagraphs>Category</CardParagraphs>
-        <h5><b>John Doe</b></h5>
-        <CardParagraphs>$123</CardParagraphs>
+      <CardParagraphs>{props.category}</CardParagraphs>
+        <h5><b>{props.name}</b></h5>
+        <CardParagraphs>{props.salePrice}</CardParagraphs>
+        <CardParagraphs>${props.price}</CardParagraphs>
```

Comment:
Be consistent with your code. I don't think that you need a dollarsign on line 43 because JSX will interpret curly brackets to mean that it is a variable.

Reply:
The $ is to have the price render $0.00 instead of 0.00. great catch. I had to check again.